plugins/org.eclipse.xtext.xbase/org.eclipse.xtext.xbase.GenerateXbase.launch

```diff
@@ -8,7 +8,9 @@
 </listAttribute>
 <stringAttribute key="org.eclipse.debug.ui.ATTR_CONSOLE_ENCODING" value="ISO-8859-1"/>
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
+<stringAttribute key="org.eclipse.jdt.launching.CLASSPATH_PROVIDER" value="org.eclipse.m2e.launchconfig.classpathProvider"/>
```

Comment:
fixed in d25403b5bcafcda0c304113e89304b2aca8b50b9
thx for the pointer, that file was not intended to go into the commit.